src/VisualStudio/Core/Def/Implementation/Options/RoamingVisualStudioProfileOptionPersister.cs

```diff
@@ -44,7 +44,9 @@ private class SVsSettingsPersistenceManager { };
         private readonly Dictionary<string, List<OptionKey>> _optionsToMonitorForChanges = new Dictionary<string, List<OptionKey>>();
         private readonly object _optionsToMonitorForChangesGate = new object();
 
-        /// <remarks>We make sure this code is from the UI by asking for all serializers on the UI thread in <see cref="HACK_AbstractCreateServicesOnUiThread"/>.</remarks>
+        /// <remarks>
+        /// We make sure this code is from the UI by asking for all <see cref="IOptionPersister"/> in <see cref="RoslynPackage.InitializeAsync"/>
```

Comment:
Thanks for fixing these comments -- I see that code hasn't actually done that for years. :-/